src/Body/Fields/Signature/Signature.js

```diff
@@ -3,14 +3,12 @@ import SignatureCanvas from 'react-signature-canvas'
 import { Custom, Input } from 'boomform'
 
 const Signature = (props) => {
-  const { id, initial, validation } = props
+  const { id, initial, validation, clearBtnContent } = props
```

Comment:
Lets add "Clear" text as a default value of clearBtnContent property and also add information about this in builder's docs .